### PR TITLE
Adquery ID System: add new ID module

### PR DESCRIPTION
**Pull request by adquery**
## Type of change

- [x] Feature

## Description of change
added adqueryID module

### CODE REVIEW

File: test/spec/modules/adqueryIdSystem_spec.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      const callbackSpy = sinon.spy();
+      const callback = adqueryIdSubmodule.getId(config).callback;
+      callback(callbackSpy);
+      expect(callbackSpy.lastCall.lastArg).to.deep.equal({qid: 'qid'});
+    });
+
+    it('allows configurable id url', function() {
+      const config = {
+        params: {
+          url: 'https://bidder.adquery.io/prebid/qid'
```
Comment (smenzer) [inline test/spec/modules/adqueryIdSystem_spec.js:44]: you should probably test with a value that isn't the default to prove that you can properly override

**Review by smenzer — CHANGES_REQUESTED** [review]
the code written so far looks good except for one comment in your tests. however, you're missing quite a few other files that need to be updated, as well as a link to the docs repo.

other files to touch include: 
- .submodules.json
- eids.js
- userId.md
- eids_spec.js
- userId_spec.js

File: test/spec/modules/adqueryIdSystem_spec.js (hunk 1 above)
Comment (adquery) [reply test/spec/modules/adqueryIdSystem_spec.js]: I added missing files and fixed test

**Review by smenzer — APPROVED** [review]
LGTM

**Comment by adquery** [comment]
Hi,

commit was merged into prebid:master on 22 Dec 2021 but I can't see  adquery module for User ID on https://docs.prebid.org/download.html
should I see it on the website already?

Best,

**Comment by adquery** [comment]
Hi,

I can't see the adquery module for User ID at https://docs.prebid.org/download.html (attached screen)
please help what else should I do to appear on the page in the "User ID Modules" section for download?

image

Thanks and best

**Pull request merged**